src/pgduckdb_background_worker.cpp

```diff
@@ -280,9 +315,12 @@ ShmemStartup(void) {
 		 */
 		MemSet(BgwShmemStruct, 0, size);
 		SpinLockInit(&BgwShmemStruct->lock);
-		BgwShmemStruct->bgw_latch = nullptr;
-		BgwShmemStruct->activity_count = 0;
-		BgwShmemStruct->bgw_session_hint_is_reused = false;
+
+		HASHCTL info;
+		info.keysize = sizeof(Oid);
+		info.entrysize = sizeof(BgwStatePerDB);
+		auto max_entries = max_worker_processes > 16 ? max_worker_processes : 16;
```

Comment:
This can use `std::max` (C++) or `Max` (Postgres C) 
```suggestion
		auto max_entries = max_worker_processes > 16 ? max_worker_processes : 16;
```